airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -683,11 +683,11 @@
 - name: Paypal Transaction
   sourceDefinitionId: d913b0f2-cc51-4e55-a44c-8ba1697b9239
   dockerRepository: airbyte/source-paypal-transaction
-  dockerImageTag: 0.1.7
+  dockerImageTag: 0.1.7.1
```

Comment:
0.1.8  ?

Reply:
this will be updated after publish automatically.